src/simplnx/DataStructure/DataPath.cpp

```diff
@@ -22,7 +22,7 @@ DataPath::DataPath(std::vector<std::string> path)
   {
     if(!DataObject::IsValidName(item) && !item.empty())
     {
-      throw std::invalid_argument("DataPath: Invalid DataObject name");
+      throw std::invalid_argument(fmt::format("DataPath: Invalid DataObject name - [{}]. One of the DataObject names contains the '/' character.", fmt::join(m_Path, ",")));
```

Comment:
This is not related to the CSV bug fix and should probably be in its own PR.

Reply:
It sort of is related since it is throwing an error about a DataObject name that is illegal. Something that the ReadCSVFileFilter could potentially throw, although not any more. But the error message is clearer as to what happened.